### PR TITLE
Add mri versions to .travis.yml

### PR DESCRIPTION
**Pull request by asymmetric**
Test against:

* 1.9.3-p551
* 2.1.6
* 2.2.2

Also, update README accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 A collection of middleware to help build services with JSON Schema.
 
+## Supported Ruby Versions
+
+Committe is tested on the following MRI versions:
```
Comment (brandur) [inline README.md]: Minor typo here. "Committee".

**Comment by brandur** [comment]
Left a minor comment, but looks good. Thanks!

**Comment by asymmetric** [comment]
Typo fixed, any chance to get this merged?

**Comment by brandur** [comment]
Oh for sure. Thank-you!

**Pull request merged**